copy.go

```diff
@@ -319,12 +320,16 @@ func mountOrCopyNode(ctx context.Context, src content.ReadOnlyStorage, dst conte
 					return nil, err
 				}
 			}
-			return src.Fetch(ctx, desc)
+			rc, err := src.Fetch(ctx, desc)
+			if err != nil {
+				return nil, err
+			}
+			return rc, nil
```

Comment:
This change seems unnecessary?

Reply:
Good catch, this was left from the previous iteration...